api/v1beta1/azurecluster_default.go

```diff
@@ -29,6 +29,9 @@ const (
 	DefaultControlPlaneSubnetCIDR = "10.0.0.0/16"
 	// DefaultNodeSubnetCIDR is the default Node Subnet CIDR.
 	DefaultNodeSubnetCIDR = "10.1.0.0/16"
+	// DefaultClusterSubnetCIDR is the default Cluster Subnet CIDR.
+	// TODO:- (Picked up the same as cp CIDR, need to check with capz reviewers).
```

Comment:
That's probably fine to start with as a default, if anything we could make it larger since we have all nodes in one subnet, although /16 is 65k IPs already, that should be plenty.